lib/helpers/mail/Mail.php

```diff
@@ -741,9 +741,42 @@ class Content implements \JsonSerializable
 
     public function __construct($type, $value)
     {
+        if ($type == 'text/plain') {
+            $value = $this->convertHTMLPlainText($value);
+        }
         $this->type = $type;
         $this->value = mb_convert_encoding($value, 'UTF-8', 'UTF-8');
     }
+    
+    public function convertHTMLPlainText($value) 
+    {
```

Comment:
Why is this method public?, it should be private, also, this breaks SRP, this shouldn't be responsibility of `Content`, maybe would be better to move to another object which would be responsible to handle these conversions, doing it this way, allows us freedom for better for testing.

Reply:
You're right about it being a public function is totally unnecessary. I am curious what you would do for SRP. I am not familiar with the term and had to read about it. 

I made it part of Content because thats where plain/text is inserted from the examples I gathered

`
$content = new SendGrid\Content("text/plain", "and easy to do anywhere, even with PHP");`

Perhaps in the new refactored version it will be better suited in a different object for portability. IE

`$plainTextContent = new PlainTextContent("and easy to do anywhere, even with PHP");`

For now I will make the modification to a private function.